composer.json

```diff
@@ -14,7 +14,8 @@
     ],
 
     "require": {
-        "bolt/filesystem": "^2.0",
+        "bolt/filesystem": "^2.1",
+        "contao/imagine-svg": "^0.1.2",
```

Comment:
Looked at the code, and I find it hard to believe they aren't going to keep 0.x a stable series, i.e using `^` is anything but a good idea.